app/src/main/java/com/flab/findmycat/ui/list/CatsFragment.kt

```diff
@@ -0,0 +1,57 @@
+package com.flab.findmycat.ui.list
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Toast
+import androidx.fragment.app.Fragment
+import androidx.navigation.fragment.findNavController
+import com.flab.findmycat.databinding.FragmentCatsBinding
+import com.flab.findmycat.domain.Cat
+import com.flab.findmycat.network.ResultOf
+import com.flab.findmycat.util.LoadMoreListener
+import org.koin.androidx.viewmodel.ext.android.sharedViewModel
+
+class CatsFragment : Fragment() {
+    private val viewModel: CatsViewModel by sharedViewModel()
+    private val listAdapter by lazy {
+        CatsAdapter(this, object : CatClickListener {
+            override fun onClick(cat: Cat) {
+                val action = CatsFragmentDirections.actionFragmentCatsToFragmentCatDetail(cat.id)
+                findNavController().navigate(action)
+            }
+        }, object : LoadMoreListener {
```

Comment:
```suggestion
        }, viewModel.loadMorelistener or viewModel
```
fragment에서 loadmore listener를 구현하기보단 viewModel에 해당 책임을 위임하는것이 좀 더 적절해보입니다 :)